src/UI/Settings/SettingsLayoutTemplate.hbs

```diff
@@ -11,7 +11,7 @@
 </ul>
 
 <div class="row settings-controls">
-    <div class="col-sm-4 col-sm-offset-7 col-md-3 col-md-offset-8">
+    <div class="col-xs-9 col-sm-5 col-sm-offset-6 col-md-4 col-md-offset-7 col-lg-3 col-lg-offset-8">
```

Comment:
This causes the Nexus 5x screen (412x732) to overflow horizontally and require scrolling.

Reply:
Can you sent me a screenshot of this, cause I can't reproduce it?
![screen shot 2017-11-06 at 19 38 02](https://user-images.githubusercontent.com/1887585/32457640-1b911bac-c32a-11e7-8cea-9545581df2bc.png)

Reply:
I have to admit that this does not work a 100% on even smaller screens like the iPhone 5 or older android Devices. The text `Advanced Settings` gets put underneath the slider.

Reply:
Leave this change out then, the v3 UI will change it anyways.